### ISSUE TITLE
Feature request: disable compression

### ISSUE THREAD

**Issue by daniviga** [zbackup/zbackup#64]
I'm running ZBackup (I love this tool!) also on an homemade NAS built on top of an HP MicroServer to backup small LVM volumes. In this case I have lot of free disk space for backups but a not so powerful CPU: is it possible to add an option to the ``--compression``  command line flag to disable the compression? Something like:

```bash
backup --non-encrypted --compression no backup /path/to/dir
``` 

(I'm using 1.4.1 from Fedora 21)

Thanks

**Comment by Am1GO** [comment]
Yes, it will be implemented soon.

**Comment by ivlis** [comment]
I am all for it. I am storing images in DNG format, thus the compression just wastes my  NAS's tiny CPU.

**Comment by Am1GO** [comment]
Implemented in #89
